src/language/branches.md

```diff
@@ -129,20 +129,14 @@ check out the patterns documentation on [Switch statements and expressions][].
 
 ### Switch expressions
 
-A `switch` expression is similar to a `switch` statement, but you can use them
-anywhere you can use an expression. They produce a value based on the expression
-body of whichever case matches.
+A `switch` _expression_ produces a value based on the expression
+body of whichever case matches. 
+You can use a switch expression wherever Dart allows expressions,
+_except_ at the start of an expression statement.
+Not all switch statments need to return a value into some expression statement.
```

Comment:
Yeah, this sentence doesn't quite work. I would maybe do:

---

You can use a switch expression wherever Dart allows expressions. For example:

```dart
// TODO: Real examples...
var x = switch (y) { example code here... };

print(switch (x) { example code ... };

return switch (x) { example code ... };
```

You can't use a switch expression at the start of an expression statement. Instead, use a switch statement.

Reply:
I actually like the not-real examples here, going to leave them empty just quickly show the possible structures (until someone yells at me that that's horrible practice!)